.changeset/plenty-birds-melt.md

```diff
@@ -0,0 +1,9 @@
+---
+"wrangler": patch
+---
+
+Adds the following new option for `create` and `update` commands:
```

Comment:
How about this? Just to make sure people know this is about the pipeline commands 

```suggestion
Adds the following new option for `wrangler pipelines create` and `wrangler pipelines update` commands:
```